L127-cpp-spiffe-root-provider.md

```diff
@@ -0,0 +1,53 @@
+L127: C++: SPIFFE Bundle Map support in Root Providers
+----
+* Author(s): gtcooke94
+* Approver: markdroth
+* Status: Draft
+* Implemented in: core, cpp
+* Last updated: 2025-07-15
+* Discussion at: TODO
```

Comment:
Please create a mailing list thread and add the link here, as per the gRFC process.  Note that that starts the 10-day clock before we can merge this.